kuma/wiki/models.py

```diff
@@ -1478,6 +1481,11 @@ def get_contributors(self):
         return User.objects.filter(pk__in=list(top_creator_ids))
 
 
+@receiver(render_done)
+def purge_wiki_url_cache(sender, instance, **kwargs):
+    memcache.delete(u'wiki_url:%s:%s' % (instance.locale, instance.slug))
```

Comment:
I'm reluctant to cache warm this since the render system is known to be having race conditions. The less state I add the better. Then there is also the question what "render_done" is compared to just save and delete for example. E.g. imagine a document translation that would be deleted, the URL should point at the original document from that point on, but we'd have both the parent document rendered as well as the old document being deleted. I sure this would create some kind of conflict eventually.